src/main/scad/laszlo-molnar/no-holes-barred/molnar.no-holes-barred.scad

```diff
@@ -0,0 +1,58 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+box_puzzle_top_inset = 1;
+
+$burr_inset = 0.12;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 16;
```

Comment:
I've been standardizing on a default of $burr_scale = 17 for this type of puzzle.. I suggest going with that for consistency